linkis-computation-governance/linkis-computation-governance-common/src/test/scala/org/apache/linkis/governance/common/paser/ScalaCodeParserTest.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.linkis.governance.common.paser
+
+import org.junit.jupiter.api.{Assertions, DisplayName, Test}
+
+class ScalaCodeParserTest {
+
+  @Test
+  @DisplayName("parseTest")
+  def parseTest(): Unit = {
+
+    val scalaCode =
+      "val codeBuffer = new ArrayBuffer[String]()\n    val statementBuffer = new ArrayBuffer[String]()"
+    val scalaCodeParser = new ScalaCodeParser
+    val array = scalaCodeParser.parse(scalaCode)
+    Assertions.assertTrue(array.size == 1)
+
```

Comment:
Might need to simulate more code string samples
such as 
- abnormal code
- or code with some special characters